tdei-api-gateway-stage.json

```diff
@@ -2181,24 +2562,274 @@
+                            "other_published_locations": {
+                                "type": "string",
+                                "description": "Other published locations"
+                            },
+                            "dataset_update_frequency_months": {
+                                "type": "number",
+                                "description": "Dataset update frequency in months"
+                            },
+                            "schema_validation_run": {
+                                "type": "boolean",
+                                "description": "Is schema validation run"
+                            },
+                            "schema_validation_run_description": {
+                                "type": "string",
+                                "description": "Description of schema validation run"
+                            },
+                            "allow_crowd_contributions": {
+                                "type": "boolean",
+                                "description": "Is crowd contributions allowed"
+                            },
+                            "location_inaccuracy_factors": {
+                                "type": "string",
+                                "description": "Location inaccuracy factors"
+                            }
+                        }
+                    },
+                    "dataset_detail": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Dataset name or Title that this data is known by"
+                            },
+                            "description": {
+                                "type": "string",
+                                "description": "Free text description of the data"
+                            },
+                            "version": {
+                                "type": "string",
+                                "description": "Dataset version"
+                            },
+                            "custom_metadata": {
+                                "type": "object",
+                                "description": "Custom structured JSON metadata"
+                            },
+                            "collected_by": {
+                                "type": "string",
+                                "description": "Description of who data was collected by. See Best Practices document for information on how to format this string.",
+                                "example": "See best practices document"
+                            },
+                            "collection_date": {
+                                "type": "string",
+                                "description": "date-time that data was collected",
+                                "example": "2018-02-10T09:30Z"
```

Comment:
Please keep a better example. We always got confused whether 02 is month or day? A better example would be 2019-22-10